src/inmanta/ast/statements/call.py

```diff
@@ -238,6 +238,15 @@ def call_in_context(
             except UnknownException as e:
                 result.set_value(e.unknown, self.ast_node.location)
             except UnsetException as e:
+                module: str = self.plugin.__module__
+                func: str = self.plugin.__function_name__
+                file: str = self.ast_node.location.file
+                line: str =self.ast_node.location.lnr
+                call: str = f"{module}::{func}"
+                location: str = f"({file}:{line})"
+                LOGGER.debug(
```

Comment:
I like the move. I believe this is a more appropriate place to handle the exception (though I'd like @wouterdb's input on this as well because of the fragile nature of how plugin calls are currently implemented). I think we should keep the comment referring to #2787 though. Ideally we'd move the logging [here](https://github.com/inmanta/inmanta-core/blob/703e13c3756804fe64807135b34e1663c88e092c/src/inmanta/execute/scheduler.py#L335) but that would require some exception wrapping to pass on the plugin context so this is the next best thing.